runner/src/simulate.rs

```diff
@@ -206,6 +211,9 @@ fn worker_config(target_peer_addr: String, throttle_requests: Option<usize>) ->
     config.scenario_log = "scenario.log".to_owned();
     config.transaction_log = "transaction.log".to_owned();
     config.request_log = "request.log".to_owned();
+    config.transaction_log = "transaction.log".to_owned();
+    config.scenario_log = "scenario.log".to_owned();
```

Comment:
Some of these look like duplicate lines.

Reply:
Doh, good catch. I was behind main (without realizing it somehow) and didn't notice you added it with the tokio-console changes... and apparently I botched the merge 😬.